recipe/meta.yaml

```diff
@@ -32,30 +32,30 @@ requirements:
     - dask-jobqueue
     - distributed
     - esgf-pyclient >=0.3.1
-    - esmpy !=8.1.0
+    - esmpy !=8.1.0,<8.6.0  # https://github.com/SciTools-incubator/iris-esmf-regrid/pull/342#issuecomment-2092921514
```

Comment:
Maybe we can avoid this pin here for easier future Python 3.12 compatibility? After all, as far as I've understood it, we're pinning esmpy because some version of iris-esmf-regrid is incompatible with it, and not the ESMValCore code? we can still apply the pin in ESMValTool so the recipe test results remain valid.

Reply:
yes! Was about to say the same - there are changes in iris-esmf-regrid that warrant at >=6.6 pin, but we still don't know if those changes work or not for lower versions - nothing with Core though; note that NCL still wants <8.6 though, so we'll have to relax it in Tool too, at some point

Reply:
Done in 01739ee 😊

Reply:
Thanks!